frontend/cvat/src/assets/engine/js/3rdparty/mousetrap.js

```diff
@@ -0,0 +1,1044 @@
+/*global define:false */
```

Comment:
Is this being used?

Reply:
I believe so, I think it's for keyboard shortcuts

Reply:
When I removed this file, I could no longer build my project